<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>AirdropPage</class>
    <widget class="QWidget" name="AirdropPage">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>900</width>
                <height>457</height>
            </rect>
        </property>
        <property name="windowTitle">
            <string>Airdrop</string>
        </property>
        <property name="styleSheet">
            <string notr="true"/>
        </property>


        <layout class="QVBoxLayout" name="topLayout">
            <item>
                <layout class="QHBoxLayout" name="horizontalLayout" stretch="1">
                    <item>
                        <layout class="QVBoxLayout" name="verticalLayout_2">
                            <item>
                                <widget class="QFrame" name="frame">
                                    <layout class="QVBoxLayout" name="verticalLayout_4">
                                        <item>
                                            <layout class="QGridLayout" name="gridLayout">
                                                <property name="spacing">
                                                    <number>12</number>
                                                </property>
                                                <item row="0" column="0">
                                                    <widget class="QLabel" name="label_5">
                                                        <property name="font">
                                                            <font>
                                                                <family>Open Sans,sans-serif</family>
                                                                <pointsize>16</pointsize>
                                                                <weight>75</weight>
                                                                <bold>true</bold>
                                                            </font>
                                                        </property>
                                                        <property name="styleSheet">
                                                            <string notr="true"/>
                                                        </property>
                                                        <property name="text">
                                                            <string>2XDN Airdrop</string>
                                                        </property>
                                                    </widget>
                                                </item>
                                                <item row="1" column="0">
                                                    <spacer name="horizontalSpacer_2">
                                                        <property name="orientation">
                                                            <enum>Qt::Horizontal</enum>
                                                        </property>
                                                        <property name="sizeHint" stdset="0">
                                                            <size>
                                                                <width>40</width>
                                                                <height>20</height>
                                                            </size>
                                                        </property>
                                                    </spacer>
                                                </item>
                                                <item row="2" column="0">
                                                    <widget class="QLabel" name="ethAddressLabel">
                                                        <property name="text">
                                                            <string>Enter your ETH address:</string>
                                                        </property>
                                                    </widget>
                                                </item>
                                                <item row="3" column="0">
                                                    <widget class="QLineEdit" name="ethAddressBox">
                                                        <property name="inputMask">
                                                            <string comment="ETH address" extracomment="ETH address"/>
                                                        </property>
                                                        <property name="placeholderText">
                                                            <string comment="ETH address" extracomment="ETH address"/>
                                                        </property>
                                                    </widget>
                                                </item>
                                                <item row="4" column="0">
                                                    <widget class="QPushButton" name="enrollButton">
                                                        <property name="text">
                                                            <string>Sign up for 2XDN airdrop</string>
                                                        </property>
                                                    </widget>
                                                </item>
                                                <item row="5" column="0">
                                                    <widget class="QLabel" name="processing">
                                                        <property name="text">
                                                            <string/>
                                                        </property>
                                                    </widget>
                                                </item>
                                            </layout>
                                        </item>
                                    </layout>
                                </widget>
                            </item>
                            <item>
                                <spacer name="verticalSpacer">
                                    <property name="orientation">
                                        <enum>Qt::Vertical</enum>
                                    </property>
                                    <property name="sizeHint" stdset="0">
                                        <size>
                                            <width>20</width>
                                            <height>40</height>
                                        </size>
                                    </property>
                                </spacer>
                            </item>
                        </layout>
                    </item>
                </layout>
            </item>
        </layout>

    </widget>
    <resources/>
    <connections/>
</ui>
